docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
+
+>  **Note**
+>
+> As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API (in application), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/updatecontrol.admin</privilege>
+   </privileges>
+```
+
+2. To use the functions and data types of the update control API, include the `<update_control.h>` header file in your application:
+
+```cpp
+#include <update_control.h>
+```
+
+3. If some initialization is required to use update control functions, you can use `update_control_initialize()`:
+
+```cpp
+int error_code;
+
+error_code = update_control_initialize();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+Likewise, there is a function `update_control_deinitialize()` for deinitialization.
+
+## Connecting to Server
+
+The new version of platform is usually uploaded at the server and each device must connect to server to get the new version.
+In some cases, connecting to server might require the procedure for getting the account information. It can be done by `update_control_initialize()` or at the each update operations (for example, **Checking the Latest Version**, **Downloading and Updating to New Version**).
+
+## Checking the Latest Version
+
+The device can query the latest version of current platform to the server. It can be implemented as the following:
+
+```cpp
+int error_code;
+
+error_code = update_control_check_new_version();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+
+If the newer version of platform is available, the value of `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` will be set properly
+(Its type and value depend on the platform developer).
```

Comment:
@sabeena21 , please suggest more trimmed sentence for this, not using parenthesis.

Reply:
Thanks @sabeena21. I think it's much better.

Reply:
@mijin85cho @sabeena21 It seems a little ambiguous because the result value `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` will be set by platform, not by application. The application only requests it to platform and reads it through the other API (update_control_get_property()) after.

So, is it okay to change like below?
*... available, the correct value and type for `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` that depends on the platform developer will be set.*

Reply:
@SunmLee , okay, thanks for explanation.
Sure you can change the sentence as suggested by you.